docs/design/2020-03-12-invisible-index.md

```diff
@@ -0,0 +1,91 @@
+# Proposal: Invisible index
+
+- Author(s):     [Deardrops](https://github.com/Deardrops), [wjhuang2016](https://github.com/wjhuang2016)
+- Last updated:  Mar. 12, 2020
+- Discussion at: https://github.com/pingcap/tidb/issues/9246
+
+## Abstract
+
+MySQL supports [Invisible indexes](https://dev.mysql.com/doc/refman/8.0/en/invisible-indexes.html); that is, indexes that are not used by the optimizer. 
+
+This is a useful feature when you want to drop an index in a safe way. Invisible indexes make it possible to test the effect of removing an index on query performance, without making a destructive change that must be undone should the index turn out to be required. Dropping and re-adding an index can be expensive for a large table, whereas making it invisible and visible are fast, in-place operations.
```

Comment:
`should the index turn out to be required` ?

Reply:
This statement is copied from [mysql docs](https://dev.mysql.com/doc/refman/8.0/en/invisible-indexes.html), translating into Chinese:
```
不可见的索引可以测试移除索引对查询性能的影响，而不会做出破坏性的更改，如果（移除后）发现索引是必需的，则必须撤销这些更改。
```